tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -1,4 +1,4 @@
-/*
+CONFIG_BOARD_BMD340EVAL_NRF52840/*
```

Comment:
This doesn't look right. Rebase mistake?

Reply:
No, copy/paste mistake. That should be fixed now.